attributes/network/14_proof.adoc

```diff
@@ -0,0 +1,28 @@
+= Proof Attribute (#14)
+:cddl: ./cddl/
+
+This attribute allows server to deliver the proof of an object requested.
+The proof itself is a sequence of actions used to construct the state in which the object is embedded within the corresponding Merkle tree.
+
+== Attribute Argument
+
+This is a uint representing the hasher to be used in constructing proofs.
+
+* 0 - merk default hash (currently blake3)
+
+* 1 - TBD
+
+== Hash Scheme
```

Comment:
Describing a Merkle tree and how to calculate the proof is fine, but we should separate the Merkle tree part from the description of the hashing itself.